exporters/otlp/otlp.go

```diff
@@ -231,3 +239,51 @@ func (e *Exporter) uploadTraces(ctx context.Context, sdl []*tracesdk.SpanData) e
 	}
 	return err
 }
+
+func NewExportPipeline(ctx context.Context, exportOpts []ExporterOption,
+	resOpts []resource.Option) (*Exporter, *sdktrace.TracerProvider, *push.Controller, error) {
+
+	exp, err := NewExporter(ctx, exportOpts...)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	res, err := resource.New(ctx, resOpts...)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	bsp := sdktrace.NewBatchSpanProcessor(exp)
+	tracerProvider := sdktrace.NewTracerProvider(
+		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+		sdktrace.WithResource(res),
+		sdktrace.WithSpanProcessor(bsp),
+	)
+
+	pusher := push.New(
+		basic.New(
+			simple.NewWithExactDistribution(),
```

Comment:
I'm curious what others think, but reading the descriptions of the available distributions I'd suggest using the inexpensive one by default in the installer pipeline.